doc/pairs.md

```diff
@@ -0,0 +1,205 @@
+# Pairs examples
+
+With ``crud.pairs``, you can iterate across a distributed space. The arguments are the same as [``crud.select``](https://github.com/tarantool/crud/docs/select.md), except or the ``use_tomap`` parameter. Below are examples that may help you.
```

Comment:
Please split this line into several lines. Currently it's too long.

Also `except or` -> `except of`?